src/protagonist/Engine/Ingest/Image/ImageServer/ImageServerClient.cs

```diff
@@ -227,6 +222,14 @@ void SetAssetLocation(ObjectInBucket objectInBucket)
             return;
         }
 
+        if (processorFlags.AlreadyUploaded)
+        {
+            // file has been uploaded already, and no image is required
+            logger.LogDebug("Asset {AssetId} has been uploaded already from the file channel, and no image delivery channel has been specified", context.AssetId);
+            SetAssetLocation(context.StoredObjects.Keys.First());

```

Comment:
I did this based on this - https://github.com/dlcs/protagonist/blob/develop/src/protagonist/Engine/Ingest/WorkerBuilder.cs#L40, but yes it's entirely possible it could change in the future